platforms/jvm/language-java/src/integTest/groovy/org/gradle/api/tasks/JavaToolchainBuildOperationsIntegrationTest.groovy

```diff
@@ -556,6 +556,83 @@ class JavaToolchainBuildOperationsIntegrationTest extends AbstractIntegrationSpe
         output.contains(jdkMetadata.javaHome.toString())
     }
 
+    @Requires(UnitTestPreconditions.Unix)
+    def "emits toolchain usages for JavaExec task with configured executable that cannot be probed"() {
```

Comment:
👍  It's good that we capture this explicitly.

Having `unknown` for all Toolchain properties in a Build Scan is not ideal, but we can improve it in a follow-up.